### DISCUSSION TITLE
when accessing the active product, etsy says "Sorry, this item is unavailable."

### DISCUSSION THREAD

**Discussion by namlv94** [etsy/open-api#884] [Q&A]
We have products on etsy with id 1412362826 in active state. However when accessing the product via the link https://www.etsy.com/listing/1412362826/original-acier, etsy says "Sorry, this item is unavailable.". This is the first time I've encountered this and am still confused as to why this is the case. Please help me to solve this problem. Thanks a lot!

**Comment by emergingdzns** [comment]
@namlv94 It's possible that the inventory is off on this listing or maybe it's not set to be viewable in the US or something. It's not an API related problem so I'm not sure this is the right forum for asking the question. I'd suggest contacting support.
